conbench/.flake8

```diff
@@ -0,0 +1,19 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
From a Python perspective, it's probably a mistake to name this directory `conbench`. Thoughts on what you would like `arrow-datafusion/conbench/` to be named?

- `arrow-datafusion/_conbench/`?
- `arrow-datafusion/conbench-benchmarks/`?